dev/templates/error-messaging.md

```
@@ -0,0 +1,74 @@
+# Error Messaging Pattern
+
+Standard pattern for tree-style error output in horizons.
+
+## Structure
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)